scripts/upgrade-via-npm.ps1

```
@@ -96,365 +166,801 @@ if (-not $CMD) {
+    if (Test-Path $CONFIG_FILE) {
+        try {
+            $cfg = Get-Content $CONFIG_FILE -Raw | ConvertFrom-Json
+            foreach ($k in @("qqbot", "openclaw-qqbot", "openclaw-qq")) {
+                if ($cfg.channels.$k) {
+                    if ($cfg.channels.$k.token) { $current = $cfg.channels.$k.token; break }
+                    if ($cfg.channels.$k.appId -and $cfg.channels.$k.clientSecret) {
+                        $current = "$($cfg.channels.$k.appId):$($cfg.channels.$k.clientSecret)"; break
+                    }
+                }
             }
+        } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)